internal/progress/progress_test.go

```diff
@@ -92,7 +92,6 @@ func TestEndProgress(t *testing.T) {
 
 func TestEndProgressTwice(t *testing.T) {
 	output := lsp.ProgressParams{
-		Token: "token",
```

Comment:
Why is this removed?

Reply:
`govet` noticed it's not used:

```
internal/progress/progress_test.go:95:8: unusedwrite: unused write to field Token (govet)
                Token: "token",
                     ^
```

Reply:
Yeah, that's the problem with linting on tests - a correct parameter must contain it, but in this test it is not checked.